resources/profiles/Anker/process/0.25mm Fast @Anker.json

```diff
@@ -13,8 +13,8 @@
     "top_shell_layers": "3",
     "support_bottom_z_distance": "0.25",
     "support_top_z_distance": "0.25",
-    "top_surface_pattern": "rectilinear",
-    "bottom_surface_pattern": "rectilinear",
+    "top_surface_pattern": "zig-zag",
```

Comment:
It's fine as  the applciation will automatically fall back to the default patten.
It's still nice to correct it anyway.

But I would suggest we use `monotonic` for these

Reply:
Changed as suggested. I was trying to shave off time, but it is so minimal.